src/docs/implementations/statsD.adoc

```diff
@@ -64,6 +64,26 @@ MeterRegistry registry = StatsdMeterRegistry.builder(StatsdConfig.DEFAULT) <2>
 1. Define what to do with lines.
 2. The flavor configuration option determines the structure of the line for the default line builder. Has no effect if you override the line builder with a customization.
 
+== Using Apache Kafka for line sink
```

Comment:
I think this should be under the section for `Customizing the metrics sink` since this is just another example of a way you can customize it.
```suggestion
=== Using Apache Kafka for line sink
```

Reply:
@shakuzen Thanks for the feedback! I pushed d076ed9 as suggested.